resources/usa_toggleclothes/sv_toggleclothing.lua

```diff
@@ -38,6 +38,11 @@ TriggerEvent('es:addCommand', 'bag', function(source, args, char, location)
 	TriggerClientEvent("headprops:toggleComponent", source, 5)
 end, { help = "Take your bag on / off." })
 
+-- Toggles backpacks --
```

Comment:
Sign of a copy pasta right here: 
```suggestion
-- Toggles shoes--
```